src/components/DashboardTabs/DashboardTabs-container.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import Grid from '@material-ui/core/Grid';
+import DashboardTabsView from './DashboardTabs-view';
+import { useDashboardStyles } from './DashboardTabs-styles';
+
+const DashboardTabs = () => {
+  const classes = useDashboardStyles();
+
+  const tabsList = ['Buying', 'Selling', 'Data'];
```

Comment:
Czy ta lista będzie jeszcze gdzieś wykorzystywana? Jeśli nie, to OK. A jeśli tak, to może warto by ją do jakichś constów przenieść (?)

Reply:
@agatascalac na chwile obecną nie, zobaczymy jak będziemy wszystko podpinać i jeśli będzie to wykorzystywane w wielu miejscach to przeniesiemy 😉